README.rst

```diff
@@ -16,6 +16,20 @@ Features
 
 * TODO
 
+PyPI Package Release
+--------------------
+- This reposiroty publishes package on PyPI, when you update this repository make sure to create a new release so latest changes should be published on PyPI and become available for your use.
```

Comment:
nits: reposiroty => repository.   "package" => "packages" or "a package".  "should" => "would"

Reply:
oh, may bad, so habitual of auto correction